.github/workflows/test.yml

```diff
@@ -83,6 +84,9 @@ jobs:
               if: ${{ matrix.experimental }}
               run: composer require "laravel/laravel:${{ matrix.laravel }}" "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update --ignore-platform-reqs
 
+            - name: Finish dependencies
+              run: composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
```

Comment:
I don't think it's a good idea to do update here. Might be better to move dependency version flag down to install. Since we don't have a `.lock` file, we always get newest packages while installing.